discovery-provider/plugins/pedalboard/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
apples to apples, want this config to look similar to our other ones, e.g. https://github.com/AudiusProject/audius-protocol/blob/main/eth-contracts/.eslintrc.js